eosdis-nasa/earthdata-pub-api
# Edpub 1331 email action

# Description

Adds a mock action for emailing DAAC Staff in the ORNL workflow, Updates sendEmail functionality to use custom templates, updates database user.findAll query to take a list of return fields

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1331


---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Deploy branch to SIT.
3.  Ensure you have ORNL DAAC Staff as a role
4. Create a new request for the ORNL DAAC in SIT
5. Follow through form and verify you receive an email that says

> The submission <submission name> has reached the email DAAC step of the ORNL Default Workflow.
